### PR TITLE
[4.1] Unnecessary description for layout

### PR DESCRIPTION
**Pull request by Kostelano**
Pull Request for Issue # .

### Summary of Changes
In 95% of modules for the control panel, the description for the LAYOUT parameter was previously removed.

Screenshot_1

The PR removes the last 2.

In the "Login Support Information" module, the "Automatic Title" parameter has been moved 1 param lower, as it corresponds to the position in the other modules.

### CODE REVIEW

**Comment by HLeithner** [comment]
Please deprecate the language string and don't remove it please

**Comment by Kostelano** [comment]
@HLeithner, so will it be right (copy/paste from 3.10)?

`; Deprecated, will be removed with 5.0. Please do not translate the following language string`

**Comment by brianteeman** [comment]
Please NO

**Comment by Kostelano** [comment]
> Please NO

Be specific. What exactly is NO?

**Comment by brianteeman** [comment]
please no for marking it as deprecated

**Comment by HLeithner** [comment]
> please no for marking it as deprecated

what else would you like to do with the string?

**Comment by brianteeman** [comment]
if its not being used then remove it. which is exactly whats happened to date in j4

**Comment by HLeithner** [comment]
> if its not being used then remove it. which is exactly whats happened to date in j4

> 6.1.6 Language keys
Changing or deleting a language key is considered a backwards compatibility break. Adding new ones is not. Substantially changing the meaning associated with a language key is a compatibility break. Rephrasing something for a more accurate description or proper en-GB grammar is not.

**Comment by Kostelano** [comment]
I changed the PR as requested above. Although, of course, it is strange to apply rule 6.1.6 only to some PRs.

Considering that 3.x modules could theoretically work on 4.x, which uses a lot of `_DESC` strings (including this one), it would probably be more correct.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7c6eba3f0344793ce5016669ca6d05d3d2565ef8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37564">issues.joomla.org/tracker/joomla-cms/37564</a>.</sub>

**Comment by Shubhamverma2796** [comment]
I have tested this item :white_check_mark: successfully on 7c6eba3f0344793ce5016669ca6d05d3d2565ef8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37564">issues.joomla.org/tracker/joomla-cms/37564</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37564">issues.joomla.org/tracker/joomla-cms/37564</a>.</sub>

File: administrator/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -202,6 +202,7 @@ JFIELD_ALIAS_LABEL="Alias"
 JFIELD_ALIAS_PLACEHOLDER="Auto-generate from title"
 JFIELD_ALT_COMPONENT_LAYOUT_DESC="Use a layout from the supplied component view or overrides in the templates."
 JFIELD_ALT_LAYOUT_LABEL="Layout"
+; Deprecated, will be removed with 5.0. Please do not translate the following language string
```
Comment (laoneo) [inline administrator/language/en-GB/joomla.ini]:
```suggestion
; Deprecated, will be removed with 5.0
```

File: api/language/en-GB/joomla.ini [hunk 2]
```diff
@@ -201,6 +201,7 @@ JFIELD_ALIAS_LABEL="Alias"
 JFIELD_ALIAS_PLACEHOLDER="Auto-generate from title"
 JFIELD_ALT_COMPONENT_LAYOUT_DESC="Use a layout from the supplied component view or overrides in the templates."
 JFIELD_ALT_LAYOUT_LABEL="Layout"
+; Deprecated, will be removed with 5.0. Please do not translate the following language string
```
Comment (laoneo) [inline api/language/en-GB/joomla.ini]:
```suggestion
; Deprecated, will be removed with 5.0

```

File: api/language/en-GB/joomla.ini [hunk 3]
```diff
@@ -1,933 +1,934 @@
...
-; States assets translations
-ARCHIVE="Archive"
-ARCHIVED="Archived"
-PUBLISH="Publish"
-PUBLISHED="Published"
-TRASH="Trash"
-TRASHED="Trashed"
-UNPUBLISH="Unpublish"
-UNPUBLISHED="Unpublished"
+; Joomla! Project
```
Comment (laoneo) [inline api/language/en-GB/joomla.ini]: What went wrong with the last commit in this file?

Comment (Kostelano) [reply api/language/en-GB/joomla.ini]: I noticed that this sometimes happens if you make adjustments through the browser.
I re-uploaded the file, everything is fine now.

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**